contracts/cw1-subkeys/Cargo.toml

```diff
@@ -19,6 +19,7 @@ library = []
 test-utils = []
 
 [dependencies]
+cosmwasm-schema = { version = "1.0.0" }
```

Comment:
We don't want this in a normal build. Technically it would probably get optimized out, but still would prolong compilation and is not wanted there in general. The workaround for this is probably to:
* make this dependency optional
* add a feature `schema-bin` enabling this dependency
* add a `[[bin]]` section like:
```
[[bin]]
name = "schema"
path = "src/bin/main.rs"
required-features = ["schema-bin"]
```

This actually might be a good reason to extract the dependency to examples... I mean it still seems better as a binary, but it requires more boilerplate in `Cargo.toml`

Reply:
It definitely gets optimized out - I tested it before.

This change is coming anyway. I already moved the schema dependency to regular deps in `cosmwasm-std 1.1.0` contracts in this PR: https://github.com/CosmWasm/cosmwasm/pull/1339 I also mentioned this problem in the PR, but didn't receive feedback against it or suggested solutions.

I know this could still be changed by adding a feature flag and doing something like:

```rs
#[cfg_attr(feature = "schema", derive(QueryResponses))]
pub enum QueryMsg {
    #[cfg_attr(feature = "schema", returns(VerifierResponse))]
    Verifier {},
    ...
}
```

But I'm not sure I'm really sold on this. Is the extra complexity worth better compilation times?

Reply:
I am mostly worried about some garbage left in the output Wasm binary so it increases in size. Did you check if wasm binary size after this change is not affected? If so then build times are secondary and it's mergeable. I am not happy about increasing complexity of dependency tree, but also its not a huge problem (at least yet).

Reply:
AFAIR, I checked the size of the Wasm binary after running it through `rust-optimizer` and it was identical to `main`, but I'm happy to double-check it before merging this.